aggregator_api/src/lib.rs

```diff
@@ -113,13 +117,23 @@ async fn auth_check(conn: &mut Conn, State(cfg): State<Arc<Config>>) -> impl Han
         constant_time::verify_slices_are_equal(bearer_token.as_ref(), key.as_ref()).is_ok()
     }) {
         // Authorization succeeds.
+        conn.set_state(cfg);
         return None;
     }
 
     // Authorization fails.
     Some((Status::Unauthorized, Halt))
 }
 
+async fn get_config(_: &mut Conn, State(config): State<Arc<Config>>) -> Json<AggregatorApiConfig> {
+    Json(AggregatorApiConfig {
+        dap_url: config.public_dap_url.clone(),
+        role: AggregatorRole::Either,
+        vdafs: vec![VdafId::Prio3Count, VdafId::Prio3Sum, VdafId::Prio3Histogram],
```

Comment:
I don't think this is right:

- it's missing `Prio3CountVec` and `Prio3SumVec`
- it needs to include `[Prio3FixedPoint16BitBoundedL2VecSum, Prio3FixedPoint32BitBoundedL2VecSum, Prio3FixedPoint64BitBoundedL2VecSum]` when the `fpvec_bounded_l2` feature is enabled

You're going to point out that several of those VDAFs don't have assigned IDs in any protocol draft, which is a good point. The point of this PR is to set up the plumbing for advertising capabilities, so I filed #1654.

Reply:
several of those VDAFs don't have assigned IDs in any protocol draft 😁